metagpt/environment/api/env_api.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# @Desc   :  the environment api store
+
+from typing import Callable
+
+from pydantic import BaseModel, Field
+
+
+class EnvAPIAbstract(BaseModel):
+    """api/interface summary description"""
+
+    api_name: str = Field(default="", description="the api function name or id")
+    args: set = Field(default={}, description="the api function `args` params")
+    kwargs: dict = Field(default=dict(), description="the api function `kwargs` params")
```

Comment:
There seems to be a need for more information here, such as desc, which can be used to describe the API, which can also come from comments and explorations, and output_type is also required

Reply:
Further, it seems that each parameter should have a corresponding comment?

Reply:
This class isn't actually bound to EnvAPIRegistry, how does it work?

Reply:
> This class isn't actually bound to EnvAPIRegistry, how does it work?

EnvAPIAbstract usage can see the `tests/metagpt/environment/test_base_env.py`